WireAnalytics/Package@swift-5.10.swift

```diff
@@ -1,76 +0,0 @@
-// swift-tools-version: 5.10
-
-import Foundation
-import PackageDescription
-
-let package = Package(
```

Comment:
removing this file does not seem to be part of this PR 🤔

Reply:
fixed, thanks